ops/update.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/bash
+
+CURR_REV=$(git rev-parse --verify HEAD)
+
+# TODO: Once config is moved out of the home dir, reset all contents of the repo before pulling.
+
+git pull
```

Comment:
this will overwrite the files while the bot is running for a brief moment which could cause problems. it might be worth testing with the origin HEAD instead of local, if changed then stop the bot, then pull.

Reply:
Good thought. Let me see what I can hack together...

Reply:
@ankitson Resolved?

Reply:
I think you have to fetch before running `rev-parse`, because otherwise git does not know what the latest revision on upstream is:

```
[23/12/09 20:44]ankit@desktop:~/devhome/code/aoc2/2023/rust (2023-day10)$ git rev-parse --verify HEAD@{u}
cb461af4464c9c4091f03f23bca2a059553416e5
[23/12/09 20:44]ankit@desktop:~/devhome/code/aoc2/2023/rust (2023-day10)$ g fetch
remote: Enumerating objects: 4, done.
...
   cb461af..fafdcb2  2023-day10 -> origin/2023-day10
[23/12/09 20:44]ankit@desktop:~/devhome/code/aoc2/2023/rust (2023-day10)$ git rev-parse --verify HEAD@{u}
fafdcb2989dae8f4bbd476e62f69d7a667bf38ac
```

Reply:
OK, I'll fix that tomorrow. Should have known that, but git is not a thing I grok just yet.